pkg/job/prometheus.go

```diff
@@ -103,3 +107,16 @@ func labels2Prom(labels map[string]string) string {
 	sl.Sort()
 	return strings.Join(sl, ",")
 }
+
+func hubType(i, last int) string {
+
+	if i == last {
+		if i == 0 {
+			return "first_last"
```

Comment:
This is a very cool idea. Using a regex we can match both first and last in this value.